internal/openshift/controller.go

```diff
@@ -104,6 +110,11 @@ func (c *controllerImpl) HandleBuild(o model.Object) error {
 		"name": user.Name,
 	})
 
+	if c.disabledUsers.Has(user.Name) {
```

Comment:
shouldn't this be moved to line 95, so that idlers are not created if it isn't needed?